### ISSUE TITLE
Some chars are escaped when storing PagedData, but not unescaped when retrieving

### ISSUE THREAD

**Issue by catageek** [SpongePowered/SpongeCommon#464]
The following code:

```
final ItemStack stack = game.getRegistry().createBuilder(ItemStack.Builder.class).itemType(ItemTypes.WRITTEN_BOOK).build();
stack.offer(Keys.BOOK_AUTHOR, Text.of(author));
stack.offer(Keys.DISPLAY_NAME, Text.of(title));
final Optional<PagedData> optPagedData = stack.getOrCreate(PagedData.class);
PagedData book = optPagedData.get();
List<Text> pages = new ArrayList<>();
String mytext = "#test=1";
pages.add(Text.of(mytext));
book.set(book.getValue(Keys.BOOK_PAGES).get().set(pages));
slot.set(stack);
return slot.peek().get().get(PagedData.class).get().pages().get(0).toPlain();
```

will return a string containing "\#test\=1" (the quotes are embedded in the string, i.e the size of the string is 11) when #test=1 is expected.

From what I could see, the chars are escaped in a JSON serializer.

**Comment by catageek** [comment]
The reason is that Gson is escaping for xml/html by default.

I could unescape it using: 
`StringEscapeUtils.unescapeJson(pages.get(i).toPlain()).replace("\"", "")`

Do we leave the plugin developers deal with it or should it be done in Sponge ?

**Comment by Zidane** [comment]
@windy1 

Status update.

**Comment by Zidane** [comment]
@SpongePowered/developers 

Status update on this or close it if not an issue.

**Comment by ImMorpheus** [comment]
I think this could be closed as a duplicate/related of #509
